elementalconductor/job.go

```diff
@@ -178,13 +201,19 @@ type FileGroupSettings struct {
 	Destination Location `xml:"destination,omitempty"`
 }
 
+// AppleLiveGroupSettings define where the HLS job output should go
+type AppleLiveGroupSettings struct {
+	Destination Location `xml:"destination,omitempty"`
+}
```

Comment:
These structs (FileGroupSettings and AppleLiveGroupSettings) are the equal, why do we need both?